react/features/e2ee/components/AbstractE2EELabel.js

```diff
@@ -27,8 +27,10 @@ export type Props = {
  * @returns {Props}
  */
 export function _mapStateToProps(state: Object) {
+    const { e2ee = {} } = state['features/base/config'];
+
     return {
-        _e2eeLabels: state['features/base/config'].e2eeLabels,
+        _e2eeLabels: e2ee.labels,
```

Comment:
We need backwards compatibility for this change. See examples here: https://github.com/jitsi/jitsi-meet/blob/12139655c6427e11cdacb5002cc890d9183e7f3e/react/features/base/config/reducer.js#L222